components/badge-status/src/_mixin.scss

```diff
@@ -10,74 +10,98 @@
   @include utrecht-badge;
   @include utrecht-badge--media-query-forced-colors;
 
+  --utrecht-badge-background-color: var(--utrecht-badge-status-background-color);
+  --utrecht-badge-border-radius: var(--utrecht-badge-status-border-radius);
+  --utrecht-badge-color: var(--utrecht-badge-status-color);
+  --utrecht-badge-font-size: var(--utrecht-badge-status-font-size, inherit);
+  --utrecht-badge-font-style: var(--utrecht-badge-status-font-style);
+  --utrecht-badge-font-weight: var(--utrecht-badge-status-font-weight);
+  --utrecht-badge-line-height: var(--utrecht-badge-status-line-height);
+  --utrecht-badge-padding-block: var(--utrecht-badge-status-padding-block);
+  --utrecht-badge-padding-inline: var(--utrecht-badge-status-padding-inline);
+
+  border-color: var(--utrecht-badge-status-border-color, transparent);
+  border-width: var(--utrecht-badge-status-border-width);
+  font-family: var(--utrecht-badge-status-font-family, inherit);
   letter-spacing: var(--utrecht-badge-status-letter-spacing, inherit);
+  min-block-size: var(--utrecht-badge-status-min-block-size, 1em);
+  min-inline-size: var(--utrecht-badge-status-min-inline-size, 1em);
   text-transform: var(--utrecht-badge-status-text-transform, inherit);
 }
 
 @mixin utrecht-badge-status--danger {
-  background-color: var(--utrecht-feedback-danger-fill-background-color, hsl(0 100% 40%));
-  color: var(--utrecht-feedback-danger-fill-color, white);
+  background-color: var(--utrecht-badge-status-danger-background-color, hsl(0 100% 40%));
+  border-color: var(--utrecht-badge-status-danger-border-color);
+  color: var(--utrecht-badge-status-danger-color, white);
 }
 
 @mixin utrecht-badge-status--warning {
-  background-color: var(--utrecht-feedback-warning-fill-background-color, hsl(48 100% 50%));
-  color: var(--utrecht-feedback-warning-fill-color, white);
+  background-color: var(--utrecht-badge-status-warning-background-color, hsl(48 100% 50%));
+  border-color: var(--utrecht-badge-status-warning-border-color);
+  color: var(--utrecht-badge-status-warning-color, white);
 }
 
 @mixin utrecht-badge-status--safe {
-  background-color: var(--utrecht-feedback-safe-fill-background-color, hsl(90 30% 50%));
-  color: var(--utrecht-feedback-safe-fill-color, white);
+  background-color: var(--utrecht-badge-status-safe-background-color, hsl(90 30% 50%));
+  border-color: var(--utrecht-badge-status-safe-border-color);
+  color: var(--utrecht-badge-status-safe-color, white);
 }
 
 @mixin utrecht-badge-status--neutral {
-  background-color: var(--utrecht-feedback-neutral-fill-background-color, black);
-  color: var(--utrecht-feedback-neutral-fill-color, white);
+  background-color: var(--utrecht-badge-status-neutral-background-color, black);
```

Comment:
Neutral: Krijgt rood kleur in plaats van zwart https://www.chromatic.com/build?appId=62d564c42f1b71308b19d073&number=3138